oak_crypto/src/encryptor.rs

```diff
@@ -230,24 +230,52 @@ pub struct ServerEncryptor {
 }
 
 impl ServerEncryptor {
-    pub fn create(
-        serialized_encapsulated_public_key: &[u8],
-        encryption_key_handle: Arc<dyn EncryptionKeyHandle>,
-    ) -> anyhow::Result<Self> {
+    /// Decrypts a [`EncryptedRequest`] proto message using AEAD.
+    /// Returns a response encryptor, the message plaintext and associated data.
```

Comment:
I'm not convinced about this API, but it is mostly pre-existing, filed #4742 to follow up.